### PR TITLE
pythonPackages.blspy: 1.0.6 -> 1.0.8

### PR DESCRIPTION
**Pull request by milahu**
works for me (TM)

### CODE REVIEW

File: pkgs/development/python-modules/blspy/default.nix [hunk 1]
```diff
@@ -40,12 +40,13 @@ buildPythonPackage rec {
...
-    # avoid dynamic linking error at import time
-    (fetchpatch {
-      url = "https://github.com/Chia-Network/bls-signatures/pull/287/commits/797241e9dae1c164c862cbdb38c865d4b124a601.patch";
-      sha256 = "sha256-tlc4aA75gUxt5OaSNZqIlO//PXjmddVgVLYuVEFNmkE=";
+      catch2_src = fetchFromGitHub {
+        owner = "catchorg";
+        repo = "Catch2";
+        rev = "v2.13.7"; # pinned by blspy
+        sha256 = "NhZ8Hh7dka7KggEKKZyEbIZahuuTYeCT7cYYSUvkPzI=";
+        fetchSubmodules = true;
```
Comment (milahu) [inline pkgs/development/python-modules/blspy/default.nix:48]: `fetchSubmodules` could be removed, maybe

Comment (milahu) [reply pkgs/development/python-modules/blspy/default.nix]: yepp, removed

**Comment by lourkeur** [comment]
doesn't break chia afaict

**Review by bbjubjub2494 — APPROVED** [review]

File: pkgs/development/python-modules/blspy/default.nix [hunk 2]
```diff
@@ -40,12 +40,12 @@ buildPythonPackage rec {
         sha256 = "sha256-lGz7o6DQVAuEc7yTp8bYS2kwjzHwGaNjugDi1ruRJOA=";
         fetchSubmodules = true;
       };
-    })
-
-    # avoid dynamic linking error at import time
-    (fetchpatch {
```
Comment (milahu) [inline pkgs/development/python-modules/blspy/default.nix:46]: todo remove `fetchpatch` from fn args

**Review by bbjubjub2494 — APPROVED** [review]

**Comment by bobby285271** [comment]
Result of `nixpkgs-review pr 150789` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>chia</li>
    <li>python38Packages.blspy</li>
    <li>python38Packages.clvm</li>
    <li>python38Packages.clvm-tools</li>
    <li>python39Packages.blspy</li>
    <li>python39Packages.clvm</li>
    <li>python39Packages.clvm-tools</li>
  </ul>
</details>

**Pull request merged**